### PR TITLE
[Mask_rom, test] Add tests for alert and otbn drivers

### PR DESCRIPTION
**Pull request by engdoreis**
Increate unittest converage for otbn and alert drivers.

### CODE REVIEW

File: sw/device/silicon_creator/lib/drivers/alert_unittest.cc [hunk 1]
```diff
@@ -39,10 +46,18 @@ TEST_F(InitTest, AlertConfigureAlertBadEnable) {
...
-  EXPECT_EQ(alert_configure(0, kAlertClassA, (alert_enable_t)-1),
+  EXPECT_EQ(alert_configure(0, kAlertClassA, static_cast<alert_enable_t>(-1)),
             kErrorAlertBadEnable);
 }
 
+TEST_F(InitTest, LocalAlertConfigureAlertBadEnable) {
+  EXPECT_SEC_WRITE32_SHADOWED(
+      base_ + ALERT_HANDLER_LOC_ALERT_CLASS_SHADOWED_0_REG_OFFSET, 0);
+  EXPECT_EQ(
+      alert_local_configure(0, kAlertClassA, static_cast<alert_enable_t>(0)),
```
Comment (alphan) [inline sw/device/silicon_creator/lib/drivers/alert_unittest.cc:57]: (just to be consistent with the test above)

```suggestion
      alert_local_configure(0, kAlertClassA, static_cast<alert_enable_t>(-1)),
```

File: sw/device/silicon_creator/lib/drivers/alert_unittest.cc [hunk 2]
```diff
@@ -126,6 +141,17 @@ TEST_F(InitTest, AlertConfigureClassXBadClass) {
   EXPECT_EQ(alert_class_configure(kAlertClassX, &config), kErrorAlertBadClass);
 }
 
+TEST_F(InitTest, AlertConfigureClassAlertBadEscalation) {
+  alert_class_config_t config{};
+  EXPECT_EQ(alert_class_configure(kAlertClassA, &config), kErrorAlertBadEnable);
+}
+
+TEST_F(InitTest, AlertConfigureClassAlertBadEnable) {
```
Comment (alphan) [inline sw/device/silicon_creator/lib/drivers/alert_unittest.cc:149]: test names should be swapped

**Review by alphan — APPROVED** [review]
Thanks @engdoreis!

File: sw/device/silicon_creator/lib/drivers/alert_unittest.cc (hunk 1 above)
Comment (engdoreis) [reply sw/device/silicon_creator/lib/drivers/alert_unittest.cc:57]: done

File: sw/device/silicon_creator/lib/drivers/alert_unittest.cc (hunk 2 above)
Comment (engdoreis) [reply sw/device/silicon_creator/lib/drivers/alert_unittest.cc]: fixed

**Review by mcy — APPROVED** [review]

**Pull request merged**